### DISCUSSION TITLE
Generate NiceGUI boilerplate UIs with a new LLM.

### DISCUSSION THREAD

**Discussion by itsmeadarsh2008** [zauberzeug/nicegui#3064] [General]
<a href="https://chat.deepseek.com/coder">DeepSeek</a>
This website uses modern datasets, so you can quickly generate a sample boilerplate for your NiceGUI application. With the help of this website, you can easily create a foundation for your application and focus on the core functionality instead of spending time on the initial setup. This modern dataset-powered site streamlines the application development process, allowing you to create high-quality applications in less time.

**Comment by falkoschindler** [comment]
Interesting, @itsmeadarsh2008.

Can you show an example of what can be generated using this tool? Who created it? How does it work?

I'm a bit hesitant to create an account for some tool I basically don't know anything about. 🫣

**Reply by itsmeadarsh2008** [reply]
It is created by DeepSeek AI. By the way, this is a trending model.

**Comment by itsmeadarsh2008** [comment]
@falkoschindler
![chat deepseek com_coder](https://github.com/zauberzeug/nicegui/assets/71959210/1b0309e3-50e0-4bff-8ab9-da7cf5cf8748)

**Comment by CVxTz** [comment]
Great find @itsmeadarsh2008 ! Thanks

**Comment by frankvp11** [comment]
Hmm, I tried it personally myself. I don't think that it uses really updated documentation, as it doesn't really know much about the current elements, and often just hallucinates about them - writing code that doesn't actually make sense in Nicegui. I'm sure it'd work great for simple apps though!
